alerts/ldap_bruteforce.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from mozdef_util.query_models import SearchQuery, TermMatch
+
+
+class AlertLdapBruteforce(AlertTask):
+    def main(self):
+        self.parse_config('ldap_bruteforce.conf', ['threshold_count', 'search_depth_min'])
+        search_query = SearchQuery(minutes=int(self.config.search_depth_min))
+        search_query.add_must([
+            TermMatch('category', 'ldap'),
+            TermMatch('details.response.error', 'LDAP_INVALID_CREDENTIALS'),
+            TermMatch('details.authenticated', 'False')
```

Comment:
Need to add a term match that requires details.user to be non-empty.  @pwnbus what's my best match object for that type of behavior?

Reply:
You can do something like:
```
search_query.add_must_not(TermMatch('details.user', ''))
```

Reply:
Thanks, added!